.craft.yml

```diff
@@ -38,6 +38,8 @@ targets:
         onlyIfPresent: /^sentry-vue-.*\.tgz$/
       'npm:@sentry/gatsby':
         onlyIfPresent: /^sentry-gatsby-.*\.tgz$/
+      'npm:@sentry/angular-ivy':
+        onlyIfPresent: /^sentry-angular-ivy.*\.tgz$/
```

Comment:
is there a - missing at the end?
also any chance the angular one might be matching this one?

Reply:
Good catch, thanks [f69929c](https://github.com/getsentry/sentry-javascript/pull/7303/commits/f69929caadf1c6e7d9eb7bd34e98de4b289c1219)